app/controllers/versions_controller.rb

```diff
@@ -0,0 +1,37 @@
+class VersionsController < ApplicationController
+  include Helpers::Geminabox
+
+  helper_method :link_to_gem, :reverse_dependencies, :load_gems
```

Comment:
Probably better, there are some annoying methods I have to "delegate" too. Will try to make that work so I don't have to include the module in the controller.